### ISSUE TITLE
Using the same encrypted realm in two apps at the same time 

### ISSUE THREAD

**Issue by MrYuDan** [realm/realm-java#6805]
E/REALM_JNI: jni: ThrowingException 5, /storage/emulated/0/AndroidWT/default.realm: Encrypted interprocess sharing is currently unsupported.SharedGroup has been opened by pid: 6583. Current pid is 6461. in /Users/cm/Realm/realm-java/realm/realm-library/src/main/cpp/io_realm_internal_OsSharedRealm.cpp line 101, .
E/REALM_JNI: Exception has been thrown: Unrecoverable error. /storage/emulated/0/AndroidWT/default.realm: Encrypted interprocess sharing is currently unsupported.SharedGroup has been opened by pid: 6583. Current pid is 6461. in /Users/cm/Realm/realm-java/realm/realm-library/src/main/cpp/io_realm_internal_OsSharedRealm.cpp line 101

**Comment by cmelchior** [comment]
Yes, this is currently not supported. It is something we want to add, but I do not have a timeline to share with you. This is currently being tracked here: https://github.com/realm/realm-core/issues/1845

I actually thought we had an Java issue tracking it, but cannot find it. So i'll promote this one to that.
